orchestration/dagster_orchestration/pyproject.toml

```diff
@@ -7,7 +7,6 @@ authors = ["Your Name <you@example.com>"]
 [tool.poetry.dependencies]
 python = "^3.9"
 dagster = "^0.10.5"
-dagit = "^0.10.5"
```

Comment:
We do not need dagit in our user code deployment, but it will still be useful to have in local dev, hence this shift.